requirements.txt

```diff
@@ -4,3 +4,4 @@
 # - tests/integration/targets/setup_botocore_pip
 boto3>=1.20.0
 botocore>=1.23.0
+jsonpatch
```

Comment:
This would also need to be added to https://github.com/ansible-collections/amazon.cloud/blob/e064de69d8a2dc62a4d4da18ebb6dd8583b07df5/README.md?plain=1#L77 Though, I'd probably just suggest removing this line because of the very reason highlighted here; it's easy for this to get out of sync with the requirements.txt.

Reply:
Removed, thanks!